srtcore/core.cpp

```diff
@@ -11147,18 +11164,23 @@ CUDTGroup::CUDTGroup(SRT_GROUP_TYPE gtype)
     switch (gtype)
     {
     case SRT_GTYPE_BROADCAST:
+        //m_bSynchOnMsgNo = false;
```

Comment:
Is this comment needed? `m_bSynchOnMsgNo`

Reply:
I'm still not sure. This was first done this way that the functions had common parts and this was used to distinguish synch on msg no to synch on seq no. Might be that this idea will be revived after some refax. I'd opt this to stay, at least as informative - or maybe to change it into a comment at this place.